pinot-plugins/pinot-input-format/pinot-clp-log/src/main/java/org/apache/pinot/plugin/inputformat/clplog/CLPLogRecordExtractorConfig.java

```diff
@@ -54,6 +55,7 @@ public class CLPLogRecordExtractorConfig implements RecordExtractorConfig {
   private final Set<String> _fieldsForClpEncoding = new HashSet<>();
   private String _unencodableFieldSuffix = null;
   private String _unencodableFieldError = null;
+  private boolean _removeProcessedFields = false;
```

Comment:
it seems the default value for removeProcessedFields is set to false. what's the current behavior, do we already by default keep the processedField e.g. message?

Reply:
The default behavior is changed. Previously, field is removed by default, now it's retained by default

Reply:
can we set removeProcessedFields=True by default to keep existing behavior. Thus, we can keep the current behavior without change tableConfig to existing tables.

Reply:
please add more comments for how to maintain the existing behavior.

Reply:
done. Added instructions on how to maintain the existing behavior via config.